live-build/config/hooks/vm-artifacts/83-etc-resolv-conf.binary

```diff
@@ -31,4 +31,4 @@
 # environment.
 #
 chroot binary rm -f /etc/resolv.conf
-chroot binary ln -s ../run/systemd/resolve/resolv.conf /etc/resolv.conf
```

Comment:
I think the `upgrade-container` script already preserves the symlink target for `/etc/resolv.conf` (it was already a symlink before, it just pointed elsewhere), right?

Reply:
So `/etc/resolv.conf` points to `/run/systemd/resolve/resolv.conf` by default. We do that step in 83-etc-resolv-conf.binary explicitly because we had changed `/etc/resolv.conf` previously.

When we do not-in-place upgrade, the initial debootstrap image will have `/etc/resolv.conf` point to `/run/systemd/resolve/resolv.conf`, which is not what we want anymore, so we have to add extra logic.

Reply:
The `upgrade-container` script preserves symbolic links, so the logic doesn't need to be modified. There is, however, a stale comment that implies that it does not do that, and that comment needs to be removed. I've done that.